stores/sql/mysql/migrations/main/schema.sql

```diff
@@ -113,35 +75,40 @@ CREATE TABLE `contracts` (
   `id` bigint unsigned NOT NULL AUTO_INCREMENT,
   `created_at` datetime(3) DEFAULT NULL,
   `fcid` varbinary(32) NOT NULL,
-  `renewed_from` varbinary(32) DEFAULT NULL,
-  `contract_price` longtext,
-  `state` tinyint unsigned NOT NULL DEFAULT '0',
-  `total_cost` longtext,
+  `host_key` varbinary(32) NOT NULL,
+
+  `archival_reason` varchar(191) NOT NULL DEFAULT '',
```

Comment:
I would neither make this `NOT NULL` nor give it a default value since this column is quite important. 
We should always be veryware of whether a contract is inserted as active or archived and a nullable column gives us very strong guarantees around that.